web/pom.xml

```diff
@@ -23,6 +23,7 @@
         <spring.version>4.3.8.RELEASE</spring.version>
         <spring.boot.version>1.5.3.RELEASE</spring.boot.version>
         <oncotree.core.version>0.0.1-SNAPSHOT</oncotree.core.version>
```

Comment:
I noticed the version of the core module is differ from the master. Is it intentional?

Reply:
@zhx828 I doubt that was intentional. Feel free to synchronize them unless we hear otherwise from @sheridancbio or @mandawilson

Reply:
I updated the version.

Reply:
@zhx828 I think all modules should get the same version number ... Yes, let's synchronize. Good catch.